.github/workflows/build.yml

```diff
@@ -266,6 +266,7 @@ jobs:
           bash .github/workflows/install_dependencies_run.sh
           ${PYTHON_EXEC} -m pip install -r requirements.txt
           rsync -am --exclude='openfpga_flow/**' /opt/openfpga/. .
+          unset OPENFPGA_PATH
```

Comment:
Do you need to add the unset to LINE 233?

Reply:
Nope its only while running from the docker image.